modules/coinswap/keeper/keeper.go

```diff
@@ -321,8 +321,47 @@ func (k Keeper) GetUniDenomFromDenoms(ctx sdk.Context, denom1, denom2 string) (s
 	if denom1 != standardDenom && denom2 != standardDenom {
 		return "", sdkerrors.Wrap(types.ErrNotContainStandardDenom, fmt.Sprintf("standard denom: %s,denom1: %s,denom2: %s", standardDenom, denom1, denom2))
 	}
+	Value := k.GetAutoIncrementId(ctx)
+	LptId, _ := strconv.Atoi(Value)
+	defer k.SetAutoIncrementId(ctx, strconv.Itoa(LptId+1))
 	if denom1 == standardDenom {
-		return fmt.Sprintf(types.FormatUniDenom, denom2), nil
+		return fmt.Sprintf(types.FormatUniDenom, strconv.Itoa(LptId+1)), nil
 	}
-	return fmt.Sprintf(types.FormatUniDenom, denom1), nil
+	return fmt.Sprintf(types.FormatUniDenom, strconv.Itoa(LptId+1)), nil
+}
+
+// SetAutoIncrementId sets Lpt AutoIncrementId for the coinswap module.
+func (k Keeper) SetAutoIncrementId(ctx sdk.Context, AutoIncrementId string) {
```

Comment:
* rename `AutoIncrementId` to `lptID`
* define lptID as uint64, serialize as follows
   ```go
   lptIDBz = make([]byte, 8)
   binary.BigEndian.PutUint64(lptIDBz, lptID)
   ```